change/@fluentui-react-native-e2e-testing-fe95f1a9-696a-45d4-8e57-ce928003e59f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Unless I'm understanding what you said wrong, I think you have "patch" and "minor" mixed up when it comes to which updates what number?

For X.Y.Z, X is the major version, Y is the minor version, and Z is the patch version. So, a "patch" will update the version from "1.0.0" to "1.0.1".

Reply:
That said, since this change is removing an exported method of BasePage, shouldn't this be a "major" change since it's a breaking change that's removes functionality that was once there? I know this package is only used by us at the moment but if this were a highly-consumed package, I think deprecating this method and making this a "minor" change before removing it in a future "major" release would be the way to go.

Reply:
So, I would add `/* @deprecated use scrollToTestElement() instead */` as a comment above `waitForPrimaryElementDisplayed()` and make this a "minor" change with comment like "deprecate waitForPrimaryElementDisplayed and remove references from tests" and we'll remove it when we bump the major version.